flake.nix

```diff
@@ -124,7 +124,10 @@
           };
           "init-ssv" = mkScriptPackage {
             name = "init-ssv";
-            deps = [nixobolus.inputs.ethereum-nix.packages."x86_64-linux".ssvnode];
+            deps = [
+              pkgs.jq
+              nixobolus.inputs.ethereum-nix.packages."x86_64-linux".ssvnode
```

Comment:
This is because the upstream has no darwin target, right? Could we make a PR upstream asking for darwin platforms?

Reply:
Exactly, I can do that. Are you referring to an issue or an actual PR?

Reply:
The compile check comes from the meta description of the package right? What about making an overlay which rewrites that. If it's ok, we can submit a PR later, but I think there must be some reason they don't want to support more platforms (possibly due CI issues, or support).